src/lock_file/resolve/pypi.rs

```diff
@@ -484,6 +487,8 @@ pub async fn resolve_pypi(
         options,
         &context.hash_strategy,
         resolver_env,
+        // TODO: This can contain tags. Which might improve the solver results on request.
```

Comment:
@tdejager, this could be an interesting feature.

Reply:
Lets investigate later indeed :)